source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Official plugin hexo-generator-tag 1.0.0, hexo-renderer-ejs 1.0.0 & hexo-renderer-stylus 1.0.0 released
+---
+
+We released a new version of the official plugins [hexo-generator-tag], [hexo-renderer-ejs] and [hexo-renderer-stylus].
+
+## hexo-generator-tag 1.0.0
+
+### Breaking change
+-  chore: require at least node 8 [#25]
+
+### Feature
+- Add the "order_by" option [#22]
+
+### Refactor
+- refactor(package): use Object.assign instead of object-assign package [#14]
+- refactor(ES2015): use let/const & arrow-return [#16)
+- refactor: es6-fy [#26](https://github.com/hexojs/hexo-generator-tag/pull/26)
+
+## hexo-renderer-ejs 1.0.0
+
+### Breaking change
+- chore: drop node 6 [#20]
+
+### Feature
+- Upgrade ejs to v2 (w/ comments support) [#5]. Support `<%# ... %>`.
+
+### Refactor
+- refactor: use Object.assign and var to let/const [#8]
+- chore(eslint): use eslint instead of jscs [#10]
+
+## hexo-renderer-stylus 1.0.0
+
+### Breaking change
+- chore: require at least node 8 [#33]
+
+### Fix
+- upgrade dependencies and make it be compatible with node v6 [#13]
```

Comment:
`upgrade dependencies and make it be compatible with node v6 [#13](https://github.com/hexojs/hexo-renderer-stylus/pull/13)` or add `[#13]: https://github.com/hexojs/hexo-renderer-stylus/pull/13` tail of markdown file.